tests/e2e/TestConstants.ts

```diff
@@ -238,12 +238,12 @@ export const TestConstants = {
     /**
      * Login for a user whom has been created in the test Openshift cluster. Need for Openshift connector test
      */
-    TS_TEST_OPENSHIFT_PLUGIN_USERNAME: process.env.TS_LOGIN_NAME_OF_OPENSHIFT_REGULAR_USER || '',
+    TS_TEST_OPENSHIFT_PLUGIN_USERNAME: process.env.TS_TEST_OPENSHIFT_PLUGIN_USERNAME || '',
 
     /**
      * Password for a user whom has been created in the test Openshift cluster. Need for Openshift connector test
      */
-    TS_TEST_OPENSHIFT_PLUGIN_PASSWORD: process.env.TS_PASSWORD_OF_OPENSHIFT_REGULAR_USER || '',
+    TS_TEST_OPENSHIFT_PLUGIN_PASSWORD: process.env.TS_TEST_OPENSHIFT_PLUGIN_PASSWORD || '',
```

Comment:
What about usage of `TS_TEST_OPENSHIFT_PLUGIN_PASSWORD` like the following?:
> TS_TEST_OPENSHIFT_PLUGIN_PASSWORD: process.env.TS_TEST_OPENSHIFT_PLUGIN_PASSWORD || process.env.TS_PASSWORD_OF_OPENSHIFT_REGULAR_USER || '',

Reply:
Sorry, I mislead `TS_PASSWORD_OF_OPENSHIFT_REGULAR_USER` propose.
No need to change anything.